tests/performance/realtime.go

```diff
@@ -80,8 +80,13 @@ var _ = SIGDescribe("CPU latency tests for measuring realtime VMs performance",
 	})
 
 	It("running cyclictest and collecting results directly from VM", func() {
-		vmi = tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskFedoraRealtime), tuneAdminRealtimeCloudInitData)
-		byConfiguringTheVMIForRealtime(vmi, "")
+		vmi = libvmi.New(libvmi.RandName(),
```

Comment:
`libvmi.RandName()`doesn't exist in main.

Reply:
Thanks. 
Strange that it compiles on my machine with latest `main`.

Reply:
Nevermind, It's probably because `make` does not compile functests.

Reply:
Yes, you need `make functests`